src/rewriter/rewrite_db_proof_cons.h

```diff
@@ -250,6 +259,8 @@ class RewriteDbProofCons : protected EnvObj
   IntStat d_statTotalAttempts;
   /** Total number of rewrites we proved successfully */
   IntStat d_statTotalInputSuccess;
+  /** Fixed point limit */
+  static size_t d_fixedPointLimit;
```

Comment:
```suggestion
  static size_t s_fixedPointLimit;
```
Should have s_ prefix since it's static.